pennylane/plugins/default_gaussian.py

```diff
@@ -754,6 +774,13 @@ def apply(self, operation, wires, par):
             self._state = self._operation_map[operation](*par, hbar=self.hbar)
             return # we are done here
 
+        if operation == 'Interferometer':
+            if par[0].shape[0] != len(wires):
+                raise ValueError("Interferomer unitary matrix applied to the incorrect "
+                                 "number of subsystems.")
+            if len(wires) > 2:
+                raise ValueError("Only 2 mode interferometers are currently supported.")
+
```

Comment:
Why are only 2 mode interferometers supported? If this is unlikely to change in the future, it would be helpful to say something like: "Please use the SF plugin if you need Interferomertes on more wires".

Reply:
Yep, solely because the `default.guassian` plugin currently only supports expanding one or two mode gates to act over the full state space (similarly to the `default.qubit` plugin).

Reply:
I'm undecided whether we should expand support here or leave the reference plugins as in, what do you think?

Reply:
I think the limited support is fine. Maybe a manual implementation of a three wire Interferometer would be useful because then one can compare a non-trivial instance to check that the numbering or BSs is compatible. Otherwise, I think referring people to the SF plugin is fine.